docs/CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Status of an async job call can be checked at endpoint: `/pub/async/task/{ID}/status`.
   See [Asynchronous calls to jobs](./user/async-job-calls.md) guide for more details.
 
+### Changed
+- Asynchronous job calls are now resilient to restarts by automatically retrying the requests,
+  The restart could be either due to an upgrade in Racetrack services
+  or a job crash (like an Out of Memory kill).
+  Regardless of the reason, the job call will keep retrying
+  until it hits the maximum number of attempts.
+  Async job calls are stored in a database for a short period.
```

Comment:
That's right. Sensitive data might be there since it keeps records of request body and response body. It actually depends which model was called and whether its requests/responses may contain sensitive data.

As a mitigation, there are 2 components making sure that a record is really deleted after "a short period". Normally, Pub service is in charge of deleting an "async job call" from a database right after retrieving it.
As a fail-safe, there's also Lifecycle Supervisor service removing periodically all obsolete records, older than 4 hours.